front/app/containers/Admin/messaging/texting/SMSCampaignPreview/index.tsx

```diff
@@ -230,13 +234,11 @@ const SMSCampaignPreview = (props: WithRouterProps) => {
         <tbody>
           <InformativeTableRow
             title="Sending to:"
-            content={`${phone_numbers.length} people`}
+            content={`${phone_numbers.length} ${phoneNumberPlural}`}
           />
           <InformativeTableRow
             title="Usage"
-            content={`${message.length} Characters (${Math.ceil(
-              message.length / 160
-            )} segments)`}
+            content={`${message.length} characters (${segmentCount} ${segmentPlural})`}
```

Comment:
Will admins understand what segments are?

Reply:
I imagine it will be part of the documentation but I'm not entirely sure. We're discussing the segment thing in Slack now so I think there will be some changes based on the outcome of that convo, so I'd just leave this for now